### DISCUSSION TITLE
Running in docker with buildkit secrets

### DISCUSSION THREAD

**Discussion by tonnyavery** [grafana/grafana#54702] [General]
There is a small snippet in documentation that explains how to run with Docker Swarm secrets: https://grafana.com/docs/grafana/v9.0/setup-grafana/configure-docker/

Unfortunately, this no longer applies for new docker Buildkit secrets as it is not possible to set global environment variables in the container with secrets.

What would be the right way to run and provision Grafana instance using docker container with new mountable secrets with Buildkit?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
